umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops-pwsh.md

```diff
@@ -0,0 +1,147 @@
+---
+description: >-
+  This section provides a step-by-step guide to setting up a CI/CD pipeline in
+  Azure DevOps using the provided sample Powershell scripts.
+---
+
+# Azure DevOps with Powershell scripts
+
+Before setting up the pipeline in Azure Devops, make sure that the steps in the [Configuring a CI/CD pipeline](./) are done.
+
+You will need to define your pipeline in YAML, and find a way to interact with the Umbraco Cloud Api.
+
+
+{% hint style="info" %}
+We have created a sample pipeline for Azure Devops.  It includes YAML-files and custom PowerShell scripts to interact with the Umbraco Cloud API.
+
+You can get the samples for Azure Devops and GitHub from this [repository](https://GitHub.com/umbraco/Umbraco.Cloud.CICDFlow.Samples).
+
+For Azure Devops you will need the following files:
+- From `powershell` folder
+  - `Get-LatestDeployment.ps1`
+  - `Get-ChangesById.ps1`
+  - `New-Deployment.ps1`
+  - `Add-DeploymentPackage.ps1`
+  - `Start-Deployment.ps1`
+  - `Test-DeploymentStatus.ps1`
+- From the `powershell/azuredevops` folder
+  - `azure-release-pipeline.yml`
+  - `cloud-sync.yml` 
+  - `cloud-deployment.yml`
+- From the root folder
```

Comment:
I think this should be the first item in the list due to the folder hierarchy

Reply:
Good point - I also did it in the github example